docs/dialect.md

```diff
@@ -174,6 +174,13 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
   **Join Capabilities**  <br/>
   Join usage is also defined by capabilities in the manifest file. See the Query section for join relation capabilities [here]({{ site.baseurl }}/docs/capabilities).
 
+  **Disable Join Modification** <br/>
+  Tableau performs modifications to join types to make queries faster. *If you use OLAP cubes for your database and want the same query as the one that is present in the cube follow this step to get the unoptimized queries:* <br/>
```

Comment:
"Tableau performs modifications to" -> "Tableau can modify"

Reply:
Thank you for the suggestion, Joe. I've updated the page to reflect the suggested changes.